### ISSUE TITLE
Custom install locations for software categories

### ISSUE THREAD

**Issue by omen124** [ChrisTitusTech/winutil#3109]
## Description
It is not associated with a problem.
When installing software from specific categories, a specific location is sometimes needed instead of the default spot.

## Solution
A button up the top or beside the category name that allows you to see an install location for installs inside a category e.g. installing game launchers into their specific folder instead of the Program Files folder.

## Alternatives
Adding an install location to the startup parameters of the script, or getting a prompt to select a location to set all installs in that instance to that location, I have no idea how to attempt this.

## Issue validation
- [X] I checked for duplicate issues.
- [X] I checked for already existing discussions.
- [X] I checked for an already existing pull request addressing the issue.

**Comment by VVilku** [comment]
Thank you for the great software. I would also be interested in the ability to specify the default directory for installation. I always do this by installing programs in a directory, e.g. 'programs', to make it easier for me to find them (and get rid of browsing everything that is installed by default in "Program Files"). How do I do this? Can I change this code somehow to add such functionality?

**Comment by brianx87** [comment]
I would also like to install my software at a custom location, I keep my software in it's own partition.

**Comment by Marterich** [comment]
TLDR: This feature will not be added in the near future

The main reason is that less than half of the packages in Winget support the --location option for custom install paths. Also, on Chocolatey, this feature is only available in the Pro version.

If we added this feature now, it could cause problems, like some programs not following the custom path. In those cases, we wouldn’t be able to help, which could be frustrating for users. So, until this feature is more widely used in Winget, we think it’s best not to add it.
/close
